code/modules/admin/admin_ranks.dm

```diff
@@ -102,7 +102,10 @@ var/list/admin_ranks = list() //list of all ranks with associated rights
 	admins.Cut()
 
 
-/hook/startup/proc/loadAdmins()
+/hook/startup/proc/hookLoadAdmins()
```

Comment:
The changes here are good but, why rename the hook proc? It doesn't make any meaningful code difference and from what I can tell, the other hooks aren't named hookX right now.